src/store/page/page-element.ts

```diff
@@ -157,69 +183,68 @@ export class PageElement {
 	 * Adds a child element to is element (and removes it from any other parent).
 	 * @param child The child element to add.
 	 * @param index The 0-based new position within the children.
+	 * @param slotId The slot to attach the element to. When undefined the default slot is used.
 	 * Leaving out the position adds it at the end of the list.
 	 */
-	public addChild(child: PageElement, index?: number): void {
-		child.setParent(this, index);
+	public addChild(child: PageElement, slotId?: string, index?: number): void {
+		child.setParent(this, slotId, index);
 	}
 
 	/**
 	 * Returns a deep clone of this page element (i.e. cloning all values and children as well).
-	 * The new clone does not have any parent.
+	 * The new clone does not have a parent nor slotId.
```

Comment:
`The new clone has neither a parent, nor a slotId.`

Reply:
Thx for the corrections :)